docs/installation.rst

```diff
@@ -0,0 +1,33 @@
+************
+Installation
+************
+
+Requirements
+============
+
+**astroplan** requires Python 2.7 or 3.3+ (2.6 and 3.2 or earlier are not
+supported) as well as the following packages:
+
+* `Numpy <http://www.numpy.org>`_
+* `Matplotlib <http://www.matplotlib.org>`_
+* `Astropy <http://www.astropy.org>`_
+* `pytz <https://pypi.python.org/pypi/pytz/>`_
+
+First-time Python users may want to consider an all-in-one Python installation
+package, such as the `Anaconda Python Distribution
+<http://continuum.io/downloads>`_ which provides all of the above dependencies.
+
+Optional packages:
+
+* `PyEphem <http://rhodesmill.org/pyephem/>`_
+* `Skyfield <http://rhodesmill.org/skyfield/>`_
+
+Installation
```

Comment:
@jberlanga - Can you write the install and testing instructions?

IMO it should be very short, i.e. I like http://wcsaxes.readthedocs.org/en/latest/installation.html and I think http://photutils.readthedocs.org/en/latest/photutils/install.html is overkill.

Mostly the install / testing instructions are the same for Astropy and every affiliated package, so instead of duplicating a lot of info there could be a link to the appropriate section on install and testing in the Astropy docs http://astropy.readthedocs.org/en/latest/ at the end.